include/seqan3/alphabet/nucleotide/dna4.hpp

```diff
@@ -132,6 +134,32 @@ struct dna4
     //!\brief The size of the alphabet, i.e. the number of different values it can take.
     static constexpr rank_type value_size{4};
 
+    /*!\name Conversion operators
+     * \{
+     */
+    //!\brief Implicit conversion between dna* and rna* of the same size.
```

Comment:
Well, it is the same documentation for rna4, dna4, rna5, dna5. I thought about hardcoding the respective other alphabet in the documentation, but then I could have hardcoded in the requires clause, as well. 
I thought this was a more generic solution that highlights that this feature is common to the nucleotides...